crtool/urls.py

```diff
@@ -20,9 +20,14 @@
     ),
     re_path(
         r'^get-review/$',
-        views.get_review,
+        views.get_review_by_id,
```

Comment:
No need to rename this. I had added a separate get_review_by_pass_code but no longer have that.